### PR TITLE
feat: Add cli command to fix eth tx hash

### PR DESCRIPTION
**Pull request by classicalliu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,23 @@ $ cat > ./packages/api-server/rate-limit-config.json <<EOF
 EOF
 ```
 
+### Fix wrong data
```
Comment (RetricSu) [inline README.md:63]: can we move fix wrong data into a new README.md under cli folder? this is a one-time fix, no need to be in the main README.md?

File: README.md [hunk 2]
```diff
@@ -60,6 +60,23 @@ $ cat > ./packages/api-server/rate-limit-config.json <<EOF
...
+
+`eth_tx_hash` was wrong before when R or S with leading zeros in database.
```
Comment (RetricSu) [inline README.md:65]: add an exact version which might contains such wrong data?

File: README.md (hunk 1 above)
Comment (classicalliu) [reply README.md]: Updated

File: README.md (hunk 2 above)
Comment (classicalliu) [reply README.md]: Updated

**Review by RetricSu — APPROVED** [review]

**Comment by keroro520** [comment]
fix-eth-tx-hash.ts is a stand-alone script. What about moving it to `packages/script/eth-tx-hash` or `packages/eth-tx-hash`?

**Comment by classicalliu** [comment]
> fix-eth-tx-hash.ts is a stand-alone script. What about moving it to `packages/script/eth-tx-hash` or `packages/eth-tx-hash`?

I want to stand with migrations. It's all for data migrate.

**Review by keroro520 — APPROVED** [review]

**Pull request merged**